llvm/lib/Target/NVPTX/NVPTXISelLowering.cpp

```diff
@@ -261,6 +261,15 @@ static void ComputePTXValueVTs(const TargetLowering &TLI, const DataLayout &DL,
     return;
   }
 
+  // Given an array type, recursively traverse the elements with custom ComputePTXValueVTs.
+  if (ArrayType *ATy = dyn_cast<ArrayType>(Ty)) {
+    Type *EltTy = ATy->getElementType();
+    uint64_t EltSize = DL.getTypeAllocSize(EltTy);
+    for (auto i : llvm::seq<int>(ATy->getNumElements()))
```

Comment:
Style nit: variables should start with a capital. Also auto is not useful here. So, `int I`.